### PR TITLE
PT-2913: Only the lower half of the logo in the header is clickable

### PR DESCRIPTION
**Pull request by sdumitriu**
Fixed

### CODE REVIEW

File: components/skin/ui/src/main/resources/PhenoTips/Skin.xml [hunk 1]
```diff
@@ -794,10 +795,14 @@ h1:target, h2:target, h3:target {
...
-  float: right;
+  color: $theme.menuLinkColor;
   cursor: default !important;
-  text-decoration: blink;
-  padding-top: 0;
+  float: right;
+  font: normal normal 700 0.8em/1.8em sans-serif;
+  margin: 0;
+  padding: 0;
+  text-transform: uppercase;
```
Comment (marta-) [inline components/skin/ui/src/main/resources/PhenoTips/Skin.xml]: We've switched all the other menus from uppercase to normal case, can we drop this line?

File: components/skin/ui/src/main/resources/PhenoTips/Skin.xml [hunk 2]
```diff
@@ -794,10 +795,14 @@ h1:target, h2:target, h3:target {
   float: right;
 }
 .language-block .rightmenu a {
-  float: right;
+  color: $theme.menuLinkColor;
   cursor: default !important;
-  text-decoration: blink;
-  padding-top: 0;
+  float: right;
+  font: normal normal 700 0.8em/1.8em sans-serif;
```
Comment (marta-) [inline components/skin/ui/src/main/resources/PhenoTips/Skin.xml]: If we get rid of uppercase (as per my comment on line `804`), this line should be  `font: normal normal 400 1em/1.4em sans-serif` to better blend into the rest of the header.

File: components/skin/ui/src/main/resources/PhenoTips/Skin.xml [hunk 3]
```diff
@@ -794,10 +795,14 @@ h1:target, h2:target, h3:target {
   float: right;
 }
 .language-block .rightmenu a {
```
Comment (marta-) [inline components/skin/ui/src/main/resources/PhenoTips/Skin.xml]: Probably not entirely relevant for this PR, but IMO "Language" should be in a `<label>` for the `<select>`, not an `<a>` element.

File: components/skin/ui/src/main/resources/PhenoTips/Skin.xml [hunk 4]
```diff
@@ -794,10 +795,14 @@ h1:target, h2:target, h3:target {
...
-  float: right;
+  color: $theme.menuLinkColor;
   cursor: default !important;
-  text-decoration: blink;
```
Comment (marta-) [inline components/skin/ui/src/main/resources/PhenoTips/Skin.xml]: Again not relevant for this PR, I just cant help but wonder _why blink_...

**Comment by sdumitriu** [comment]
@marta- Comments addressed.

File: components/skin/ui/src/main/resources/PhenoTips/Skin.xml [hunk 5]
```diff
@@ -782,24 +782,18 @@ h1:target, h2:target, h3:target {
...
 }
 .language-block .rightmenu {
   top: 10px;
   margin-right: 10px;
 }
-.language-block .rightmenu select {
-  text-transform: uppercase;
+.language-block .rightmenu label {
+  color: $theme.menuLinkColor;
+  font: normal normal 400 1em/1.4em Arial,Verdana,sans-serif;
```
Comment (marta-) [inline components/skin/ui/src/main/resources/PhenoTips/Skin.xml]: Can we use `inherit` for the font family to make it easier to change fonts in the future?

**Pull request merged**